PC/launcher2.c

```diff
@@ -871,6 +871,77 @@ _findCommand(SearchInfo *search, const wchar_t *command, int commandLength)
 }
 
 
+int
+_useShebangAsExecutable(SearchInfo *search, const wchar_t *shebang, int shebangLength)
+{
+    wchar_t buffer[MAXLEN];
+    wchar_t script[MAXLEN];
+    wchar_t command[MAXLEN];
+
+    int commandLength = 0;
+    int inQuote = 0;
+    int afterSlash = 0;
+
+    if (!shebang || !shebangLength) {
+        return 0;
+    }
+
+    wchar_t *pC = command;
+    for (int i = 0; i < shebangLength; ++i) {
+        wchar_t c = shebang[i];
+        if ((isspace(c) || c == L'\r' || c == L'\n') && !inQuote) {
```

Comment:
In the "C" locale, `isspace()` is true for 9-13 (TAB, LF, VT, FF, CR), 32 (space), and 160 (no-break space). There's no need to separately check for CR or LF.

Reply:
That's what I thought, but for some reason it didn't work. I'll try again, maybe I messed something else up.